src/screens/CreateNFT.js

```diff
@@ -0,0 +1,333 @@
+              <label>{t`Symbol`}</label>
+              <input required ref="symbol" placeholder={t`MYC (2-5 characters)`} type="text" minLength={2} maxLength={5} className="form-control" />
+            </div>
+          </div>
+          <div className="row">
+            <div className="form-group col-4">
+              <label>{t`Amount`}</label>
+              <input required type="number" min="0" step="1" className="form-control" placeholder="How many NFT units to create" onChange={this.onAmountChange} />
+            </div>
+            <div className="form-group d-flex flex-row align-items-center address-checkbox">
+              <div className="form-check">
+                <input className="form-check-input" type="checkbox" ref="autoselectAddress" id="autoselectAddress" defaultChecked={true} onChange={this.handleCheckboxAddress} />
+                <label className="form-check-label" htmlFor="autoselectAddress">
+                  {t`Select address automatically`}
+                </label>
+              </div>
+            </div>
+            <div className="form-group col-5" ref={this.address} style={{display: 'none'}}>
+              <label>{t`Destination address`}</label>
+              <input ref="address" type="text" placeholder={t`Address`} className="form-control" />
+            </div>
+          </div>
+          <div className="row mt-3">
+            <div className="form-group d-flex flex-row align-items-center create-mint-checkbox col-5">
+              <div className="form-check">
+                <input className="form-check-input" type="checkbox" ref="createMintAuthority" id="createMintAuthority" defaultChecked={false} />
+                <label className="form-check-label" htmlFor="createMintAuthority">
+                  {t`Create a mint authority`}<small id="createMintHelp" className="form-text text-muted">If you want to be able to mint more units of this NFT.</small>
+                </label>
+              </div>
+            </div>
+            <div className="form-group d-flex flex-row align-items-center create-melt-checkbox col-4">
+              <div className="form-check">
+                <input className="form-check-input" type="checkbox" ref="createMeltAuthority" id="createMeltAuthority" defaultChecked={false} />
+                <label className="form-check-label" htmlFor="createMeltAuthority">
+                  {t`Create a melt authority`}<small id="createMeltHelp" className="form-text text-muted">If you want to be able to melt the units of this NFT.</small>
+                </label>
+              </div>
+            </div>
+          </div>
+          <hr className="mb-5 mt-5"/>
+          <p><strong>HTR available:</strong> {hathorLib.helpers.prettyValue(this.props.htrBalance)} HTR</p>
+          <p><strong>Deposit:</strong> {hathorLib.helpers.prettyValue(depositAmount)} HTR</p>
+          <p><strong>Fee:</strong> {hathorLib.helpers.prettyValue(tokens.getNFTFee())} HTR</p>
```

Comment:
This was already calculated as the variable `nftFee`

Reply:
You're right, thanks. Changed here https://github.com/HathorNetwork/hathor-wallet/pull/222/commits/3544a03c03d5c88016cdda79a35b8fe4a4e50aae